dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1145,6 +1145,9 @@ static HRESULT WINAPI ddraw7_SetDisplayMode(IDirectDraw7 *iface, DWORD width, DW
                 ddrawformat_from_wined3dformat(&ddraw->primary->surface_desc.u4.ddpfPixelFormat, mode.format_id);
         }
         ddraw->flags |= DDRAW_RESTORE_MODE;
+
+		if (ddraw->cooperative_level & DDSCL_EXCLUSIVE)
+            SetWindowPos(ddraw->dest_window, HWND_TOP, 0, 0, width, height, SWP_SHOWWINDOW | SWP_NOACTIVATE);
     }
 
     InterlockedCompareExchange(&ddraw->device_state, DDRAW_DEVICE_STATE_NOT_RESTORED, DDRAW_DEVICE_STATE_OK);
```

Comment:
This patch seems to respect what I mentioned/recommended here: https://github.com/reactos/reactos/pull/4724/files#r979200669

The order of commands does match Wines order of commands.

Still the question: Should the added block be guarded with #ifdef __REACTOS until we will be on par with that Wine-version which would do the correct thing yet.

Reply:
yup a __REACTOS__ should be added